### PR TITLE
feat: added docker config

### PR DESCRIPTION
**Pull request by sashokbg**

### CODE REVIEW

File: front/src/Admin/RecapAdmin.js [hunk 1]
```diff
@@ -13,14 +13,14 @@ import { Button } from '@mui/material';
 
 
 const RecapAdmin = () => {
-  const apiUrl = 'http://localhost:3000';
```
Comment (Sofien-Touati) [inline front/src/Admin/RecapAdmin.js:16]: consider setting base url as a global const variable across the app

**Review by Sofien-Touati — CHANGES_REQUESTED** [review]

File: back/src/app.module.ts [hunk 2]
```diff
@@ -18,24 +18,43 @@ import { ProjectController } from './controllers/Project.controller';
...
-      __dirname + '/**/*.entity{.ts,.js}',
+  imports: [
+    ConfigModule.forRoot(
+      { envFilePath: dotEnvPath },
+    ),
+    TypeOrmModule.forRoot({
+      type: 'mysql',
+      host: process.env.DATABASE_HOST || 'localhost',
+      port: 3306,
+      username: 'cra_user',
```
Comment (amirakarchoud) [inline back/src/app.module.ts]: put this in .env

File: docker/docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
+services:
+
+  database:
+    image: mariadb
+    expose:
+      - 3306:3306
+    environment:
+      MARIADB_USER: root
```
Comment (amirakarchoud) [inline docker/docker-compose.yaml:8]: wrong user password

**Review by amirakarchoud — COMMENTED** [review]
add doc pls

**Pull request merged**